manifests/postgres.yaml

```diff
@@ -46,12 +46,12 @@ spec:
           initialDelaySeconds: 60
           timeoutSeconds: 5
           failureThreshold: 6
-        readinessProbe:
+        readiness-probe:
```

Comment:
Actually I do, there's no problem on parsing and changing it in runtime IMO... The closer we get to kubernetes manifests style, the best.